src/parser/SGParser.ts

```diff
@@ -49,28 +49,28 @@ export default class SGParser {
      * Walk the AST to extract references to useful bits of information
      */
     private findReferences() {
-        this._references = emptySGReferences();
+        this._references = this.emptySGReferences();
 
         const { component } = this.ast;
         if (!component) {
             return;
         }
 
         const nameAttr = component.getAttribute('name');
-        if (nameAttr?.value) {
-            this._references.name = nameAttr.value;
+        if (nameAttr?.tokens.value) {
```

Comment:
Pattern could probably be simplified into `this._references.name = nameAttr?.value;` without the `if` guard.

Reply:
Done.